Maven-Build schlägt fehl

Da ich völlig java/maven-agnostisch bin, kann ich leider nur die Fehlermeldungen beisteuern:
```
10.71 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:compile (default-compile) on project eyematics-test-data-generator: Compilation failure
10.71 [ERROR] /opt/test-data-generator/src/main/java/org/eyematics/Main.java:[56,46] cannot find symbol
10.71 [ERROR]   symbol:   method getFirst()
10.71 [ERROR]   location: interface java.util.List<org.hl7.fhir.r4.model.Bundle.BundleEntryComponent>
10.71 [ERROR]
10.71 [ERROR] -> [Help 1]
10.71 org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:compile (default-compile) on project eyematics-test-data-generator: Compilation failure
10.71 /opt/test-data-generator/src/main/java/org/eyematics/Main.java:[56,46] cannot find symbol
10.71   symbol:   method getFirst()
10.71   location: interface java.util.List<org.hl7.fhir.r4.model.Bundle.BundleEntryComponent>
10.71
10.71     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:333)
10.71     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
10.71     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
10.71     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
10.71     at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
10.71     at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
10.71     at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
10.71     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
10.71     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
10.71     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
10.71     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
10.72     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
10.72     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
10.72     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
10.72     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
10.72     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
10.72     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
10.72     at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
10.72     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
10.72     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
10.72     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
10.72     at java.lang.reflect.Method.invoke (Method.java:569)
10.72     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
10.72     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
10.72     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
10.72     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
10.72 Caused by: org.apache.maven.plugin.compiler.CompilationFailureException: Compilation failure
10.72 /opt/test-data-generator/src/main/java/org/eyematics/Main.java:[56,46] cannot find symbol
10.72   symbol:   method getFirst()
10.72   location: interface java.util.List<org.hl7.fhir.r4.model.Bundle.BundleEntryComponent>
10.72
10.72     at org.apache.maven.plugin.compiler.AbstractCompilerMojo.execute (AbstractCompilerMojo.java:1309)
10.72     at org.apache.maven.plugin.compiler.CompilerMojo.execute (CompilerMojo.java:215)
10.72     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
10.72     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
10.72     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
10.72     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
10.72     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
10.72     at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
10.72     at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
10.72     at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
10.72     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
10.72     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
10.72     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
10.72     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
10.72     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
10.72     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
10.72     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
10.72     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
10.72     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
10.72     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
10.72     at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
10.72     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
10.72     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
10.72     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
10.72     at java.lang.reflect.Method.invoke (Method.java:569)
10.72     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
10.72     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
10.72     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
10.72     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
10.72 [ERROR]
10.72 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
10.72 [ERROR]
10.72 [ERROR] For more information about the errors and possible solutions, please read the following articles:
10.72 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,7 @@
 
     <groupId>org.eyematics</groupId>
     <artifactId>eyematics-test-data-generator</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0.1</version>
 
     <properties>
         <maven.compiler.source>17</maven.compiler.source>

--- a/src/main/java/org/eyematics/Main.java
+++ b/src/main/java/org/eyematics/Main.java
@@ -53,7 +53,7 @@ public class Main {
                     .setConsentId(consentId)
                     .build();
             sharedPatient.getMeta().setSource("mailto:medic@" + d.toLowerCase() + ".org");
-            sharedPatientResources.getEntry().getFirst().setResource(sharedPatient);
+            sharedPatientResources.getEntry().get(0).setResource(sharedPatient);
             sharedPatientResources.getEntry().forEach(dicBundle::addEntry);
             bundleWriter.writeBundleToJSON(dicBundle, BASE_PATH, "testdatensatz_" + d.toLowerCase());
             System.out.println("DONE");

